src/main/java/service/ReportCreator.java

```diff
@@ -0,0 +1,22 @@
+package service;
+
+import db.Storage;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportCreator {
```

Comment:
You need to set the structure
```suggestion
public class ReportImpl implements Report {
```